src/main/java/br/codenation/hospital/manager/model/Hospital.java

```diff
@@ -51,5 +56,53 @@ public Hospital(
     this.availableBeds = availableBeds;
     this.latitude = latitude;
     this.longitude = longitude;
+    this.patients = new HashMap<>();
+    this.stock = new HashMap<>();
+  }
+
+  public Patient addNewPatient(Patient patient) {
+
+    if (!this.patients.containsKey(patient.getId())) {
+      this.patients.put(patient.getId(), patient);
+      return patient;
+    }else
+      return null;
+  }
+
+  public Patient removePatient(String patientId){
+    return this.patients.remove(patientId);
+  }
+
+  public SupplyItem addItemStock(SupplyItem supplyItem){
+
+    if(!this.stock.containsKey(supplyItem.getId())) {
+      this.stock.put(supplyItem.getId(), supplyItem);
+      return supplyItem;
+    }
+    else{
+      supplyItem.addItens(this.stock.get(supplyItem.getId()).getQuantity());
+      this.stock.replace(supplyItem.getId(),supplyItem);
+      return supplyItem;
+    }
+  }
+
+  public boolean incAvailableBeds(){
+    if(this.availableBeds < this.beds) {
+      this.availableBeds++;
+      return true;
+    }else
+      return false;
+  }
+
+  public boolean decAvailableBeds(){
```

Comment:
Doesn't need to be abbreviated.
```suggestion
  public boolean decrementAvailableBeds(){
```

Reply:
Same thing on this method